dev/bots/docs.sh

```diff
@@ -9,7 +9,7 @@ function deploy {
   local total_tries="$1"
   local remaining_tries=$(($total_tries - 1))
   shift
-  while [[ "$remaining_tries" > 0 ]]; do
+  while [[ "$remaining_tries" -gt 0 ]]; do
```

Comment:
this was a shellcheck error, not sure if this did what it was supposed to.

Reply:
`-gt` is correct.